convert.py

```diff
@@ -3,4 +3,18 @@
 
 def utf_len(string: bytes) -> int:
     """len function"""
-    return len(str(string, 'utf-8'))
+    # прибираємо лишні знаки
+    string_without_odd = str(string)[2:-1]
+    result = 0
+    index = 0
+    while index <= len(string_without_odd)-1:
+        result += 1
+        # вичисляємо кирилицю
+        if string_without_odd[index:index+3] == "\\xd":
```

Comment:
Checking of string representation of byte is not enough for correct solution. Try to use bit operators